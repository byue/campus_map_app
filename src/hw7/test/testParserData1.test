# test parser on a graph with 1 node

LoadGraph heroes data1.tsv
ListChildren heroes a
FindPath heroes a b