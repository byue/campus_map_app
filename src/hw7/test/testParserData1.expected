# test parser on a graph with 1 node

loaded graph heroes
the children of a in heroes are:
unknown character b